paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletUiState.kt

```diff
@@ -118,7 +124,11 @@ private fun ConsumerPaymentDetails.PaymentDetails.makeMandateText(
         is Card,
         is ConsumerPaymentDetails.Passthrough -> {
             if (isSettingUp) {
-                resolvableString(R.string.stripe_paymentsheet_card_mandate, merchantName)
+                if (signupToggleEnabled) {
+                    resolvableString(R.string.stripe_paymentsheet_card_mandate_signup_toggle_off, merchantName)
```

Comment:
This condition needs to be pulled out, as we need to show the mandate even if no SFU is set (which is what `isSettingUp` refers to).